website/js/components/insilicopred/InSilicoPredSubtile.js

```diff
@@ -0,0 +1,114 @@
+/*eslint-env browser */
+/*global require: false, module */
+'use strict';
+import React from "react";
+import util from "../../util";
+import KeyInline from "../KeyInline";
+import {Table} from "react-bootstrap";
+
+class PathosProbScale extends React.Component {
+    render() {
+        const {value, brackets} = this.props;
+
+        return (
+            <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100" viewBox="0 0 500 60">
+                <defs>
+                    <linearGradient id="grad1" x1="0%" y1="0%" x2="100%" y2="0%">
+                        <stop offset="0%"   style={{stopColor: '#0000FE', stopOpacity: 1}} />
+                        <stop offset="100%" style={{stopColor: '#F60010', stopOpacity: 1}} />
+                    </linearGradient>
+                </defs>
+
+                <g transform="translate(25 0)">
+                    <rect x={0} y={0} width={450} height={40} strokeWidth={2} stroke="#ccc" fill="url(#grad1)">
+                        <title>Value: {value}</title>
+                    </rect>
+
+                    {
+                        /* tickmarks */
+                        Array.from({ length: brackets + 1 }).map((x, idx) => {
+                            const val = (idx / brackets).toFixed(2);
+                            return (
+                                <text key={idx} y={70} x={(450 * val)} alignmentBaseline="hanging"
+                                    textAnchor={idx === 0 ? "start" : idx === brackets ? "end" : "middle"}>
+                                    {val}
+                                </text>
+                            );
+                        })
+                    }
+
+                    {/* indicators, if a value is given */}
+                    { value !== undefined && (
+
+                        <g>
+                            <rect x={450 * value - 2} y={4} width={4} height={32} fill="black" />
+
+                            <g transform={`translate(${450 * value} ${42})`}>
+                            <polygon stroke="none" fill="black" points="0,0 5,12 -5,12" />
+                            </g>
+                        </g>
+                    )}
+                </g>
+            </svg>
+        );
+    }
+}
+
+
+export default class InSilicoPredSubtile extends React.Component {
+    render() {
+        const {probability, reason, varLoc, varType} = this.props;
+        const summarizedData = {varLoc, varType};
+        const cols = [
+            { title: 'VEP Consequences', prop: 'varLoc' },
+            // { title: 'Variant Type', prop: 'varType' }
```

Comment:
can be removed?

Reply:
Good catch, both those sets of comments are unnecessary.